notebooks/astrokat_coords.ipynb

```diff
@@ -0,0 +1,185 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "astrokat-coords.ipynb",
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true,
+      "authorship_tag": "ABX9TyM6Ss3AciV5qgrWVJMGyLf3",
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/rubyvanrooyen/astrokat/blob/master/notebooks/astrokat_coords.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
No, this is relic output that was generated because the tests were done from a devel repo -- as soon as I use the notebooks for the next phase of devel the json will be updated

The python code that will run in the colab playground will default to ska-sa repo